tsconfig.json

```diff
@@ -4,11 +4,12 @@
     "removeComments": false,
     "preserveConstEnums": true,
     "sourceMap": true,
-    "declaration": true,
+    // "declaration": true,
```

Comment:
as same as below Q, do we need to modify tsconfig for compiler bump up? (esp. allowJS?)